packages/frontend/src/main/pages/auth/TheLogin.vue

```diff
@@ -93,6 +93,29 @@
         </div>
       </div>
     </div>
+    <div class="d-block d-md-none pa-8 pt-0">
+      <h3 class="font-weight-bold mt-0">This is the Legacy Web App</h3>
+      <p class="mb-0 mt-1 primary--text text--disabled text-body-2 pr-4">
+        A better and more powerful web app is replacing this.
```

Comment:
Ditto. 1) Validate the url 2) Validate that we want to show this to everyone

Reply:
As above, Benjamin confirmed the login page should stay as is